add support for SUBDOMAIN_INSTALL and PATH_CURRENT_SITE options

mrgum opened the pull request:
wp-config.php when used for multisite has SUBDOMAIN_INSTALL and PATH_CURRENT_SITE which I need so I've added support for them with $wp_path_current_site and $wp_subdomain_install.

Commit:
add support for SUBDOMAIN_INSTALL and PATH_CURRENT_SITE options in wp-config.php

cheethoe commented:
Is this going to get pulled in?  I think this is a critical feature for wordpress multisite support.

Commit:
when we are upgrading index.php will already exist so using it as a create is pony

Commit:
This was poorly thought through

removing the tar file would now re download extract and chmod

the exec extract needs better logic

Revert "when we are upgrading index.php will already exist so using it as a create is pony"

This reverts commit d9870d62b5e54409b752fb40ec76016dfcd58a6c.

Commit:
hackery perhaps

Commit:
Revert "hackery perhaps"

This reverts commit 463b005a420d9d2c889a5467388b4b7c4709ed36.

this was just as bad still removing the tar downloads it, then removes index.php which then sets the exec to untar it off
we need knowledge of the running version of php
maybe use wp cli program or run a curl xmlrpc call
which is least yuck

Commit:
Chamge default mode of wordpress files to 0664

This will allow members of the group $wp_group to write to the wordpress directory, enabling them to run wp-cli etc without sudo.  There shouldn't be any risk here; by default, $wp_group is '0', i.e. root

mrgum committed:
Merge pull request #1 from porcospino/patch-1

Change default mode of wordpress files to 0664

cmseal commented:
Is there any chance of this being reviewed? Looking at the CI failures, it appears related to the Ruby build rather than the PR code (I could be wrong).

mrgum commented:
life goes on, 8 years later if it isn;t pulled time to stop holding my breathe :)
